server/channels/app/slashcommands/command_test.go

```diff
@@ -424,24 +423,42 @@ func TestDoCommandRequest(t *testing.T) {
 		require.Equal(t, "api.command.execute_command.failed.app_error", err.Id)
 	})
 
-	t.Run("with a slow response", func(t *testing.T) {
+	t.Run("with a too slow response", func(t *testing.T) {
 		done := make(chan bool)
 		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			<-done
-			io.Copy(w, strings.NewReader(`{"text": "Hello, World!"}`))
+			io.Copy(w, strings.NewReader("Hello, World!"))
 		}))
 		defer server.Close()
 
-		th.App.HTTPService().(*httpservice.HTTPServiceImpl).RequestTimeout = 100 * time.Millisecond
-		defer func() {
-			th.App.HTTPService().(*httpservice.HTTPServiceImpl).RequestTimeout = httpservice.RequestTimeout
-		}()
+		th.App.UpdateConfig(func(cfg *model.Config) {
+			cfg.ServiceSettings.OutgoingIntegrationRequestsTimeout = model.NewInt64(1)
```

Comment:
q: is there a reason why we don't have  `model.NewInt8` for these small numbers?

Reply:
The reason an `int64` is used is that `time.Duration` is an `int64`, and hence it's easier to multiply.